packages/generator/src/generator-without-ts-morph/service/class.ts

```diff
@@ -54,32 +54,30 @@ export function serviceClass(service: VdmServiceMetadata): string {
     ${service.entities
       .map(entity => getEntityApiFunction(entity, service))
       .join('\n\n')}
-    
-    ${getActionFunctionImports(service, 'functionImports')}
-    
-    ${getActionFunctionImports(service, 'actionImports')}
+
+    ${getOperations(service)}
     
     ${serviceHasEntities ? getBatch() : ''}
 
     ${serviceHasEntities ? getChangeset() : ''}
   }`;
 }
 
-function getActionFunctionImports(
-  service: VdmServiceMetadata,
-  type: 'functionImports' | 'actionImports'
-): string {
-  if (service[type] === undefined || service[type]!.length === 0) {
+function getOperations(service: VdmServiceMetadata): string {
+  const operations = service.functionImports.concat(
+    service.actionImports || []
+  );
```

Comment:
For the first question, this is also because `VdmServiceMetadata` doesn't have `operations`, only `actionImports` and `functionImports`. Maybe I didn't understand the scope of the issue correctly -- should we also merge these in to `operations` in `VdmServiceMetadata` and the likes?

For the second question, the definitions of these fields in `VdmServiceEntities`:
```
  /**
   * @internal
   */
  functionImports: VdmOperation[];
  /**
   * @internal
   */
  actionImports?: VdmOperation[];
```
show that actionImports is optional. Maybe this is not intended? Then we could change that in a follow-up as well.